packages/core/src/new-api/internal/new-execution/basic-execution-strategy.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  decodeArtifactCustomError,
+  decodeArtifactFunctionCallResult,
+  encodeArtifactDeploymentData,
+  encodeArtifactFunctionCall,
+  executeOnchainInteractionRequest,
+  executeStaticCallRequest,
+} from "./execution-strategy-helpers";
```

Comment:
This should be the only import that's not types or enums that an execution strategy should need.